src/Engine/ProtoCore/FFI/CLRFFIFunctionPointer.cs

```diff
@@ -860,27 +860,6 @@ public override object Execute(ProtoCore.Runtime.Context c, Interpreter dsi, Lis
                 return null;
             }
 
-            StackValue propValue = (StackValue)retVal;
```

Comment:
so this code was updating reference types values on the heap, and we no longer do it - because it was unused?

Reply:
As far as I can tell.  I could not find where we ever retrieved this data as a alternative to un-marshaling from the DSObjectMap

Reply:
I should say un-marshaling and then calling the getter function via reflection in InvokeNoThow

Reply:
Oops you are correct about reference type vs value type....  I will to update the description.  We added the StackValue pointers not values to the heap